petclinic.yaml

```diff
@@ -11,16 +11,21 @@ tools:
     type: dockerImage
     image: maven:3.5.4-jdk-8
     entryPoint: sleep infinity
-    persistentVolumes:
+    volumes:
       - name: maven-repo
-        containerPath: /root/.m2
-    services:
+        mountPath: /root/.m2
+      - name: projects
+        mountPath: /projects
+    endpoints:
       - name: spring-boot
-        port: 8080
-    mountProjectsSources: true
+        public: true
+        targetPort: 8080
+        attributes:
+          protocol: http
+          secure: false
   - name: mysql
-    type: composeFile
-    local: docker-compose.yml
+    type: kubernetes
+    local: kubernetes-mysql.yml
```

Comment:
Adding here as well so that it is not lost 

Is `kubernetes-mysql.yml` a regular kubernetes resource manifest? I think it should be. If that is the case, we are assuming(forcing???) that manifest to only include the resources that we want to be materialized as part of the workspace. This will be limiting for most projects as we will most likely encounter a helm chart or openshift template on the project. I think we should also have `selectors` in here to select the resources that we want.
```yaml
- name: mysql
  type: kubernetes
  local: application.yml
  selector: 
     name: mysql
- name: reddit
  type: kubernetes
  local: application.yml
  selector: 
     name: reddit-cache  
```